src/Youtube.test.js

```diff
@@ -28,6 +28,12 @@ describe('YouTube', () => {
     expect(queryByAttribute('class', container, 'custom-class')).toBeDefined();
   });
 
+  it('should render an iframe with a custom class name', () => {
+    const { container } = render(<YouTube frameClassName="custom-frame-class" videoId="XxVg_s8xAms" />);
+
+    expect(queryByAttribute('class', container, 'custom-frame-class')).toBeDefined();
+  });
+
```

Comment:
The interesting part of this test would be the update since that requires custom logic.

Note to self: Since the YouTube IFrame API will replace the inner div React becomes unable to update/unmount it.
Our test should mimic this behaviour as closely as possible so we can test that the component behaves correctly despite not being managed by React.
We will need to improve the mock in order to do this correctly (separate PR).

Reply:
As this is a breaking change and we are using conventional commits, @karolisgrinkevicius please add a breaking change footer to your commit as well.
Here is a link for reference: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

Reply:
Thanks @tjallingt. I have just added one more unit test to check whether the component updates or not once frame class is changed here: 5fb1708547b3f03d24a1bb97bc26f8821e9aab7a.